### ISSUE TITLE
Flickering with Firefox open

### ISSUE THREAD

**Issue by tcorreabr** [tcorreabr/Parachute#61]
Sometimes flickering occurs when Firefox is open. If I close it, flickering stops. If anyone could get around this problem, please post here.

**Comment by barolo** [comment]
I have no such issues, although tested only with latest firefox with disabled titlebar [ 84.0], latest mesa [ 20.3.1] with amd drivers and latest plasma [5.20.4]
I had to set speed to 250 ms too for the animation of windows failing back into place to look most plausible/connected

**Comment by gobonja** [comment]
Experienced same issue, mostly when using intellij.
Cannot be sure but I think this happens when system is on "heavy" load.

Also if I change focus during flickering (e.g. open krunner), flickering stoppes for sometime.

I will try to find correct way to cause this behavior so you can properly debug.

**Comment by barolo** [comment]
Oh, I've just noticed it but it doesn't affect the app itself, in my case it's the frame of selected  [ if you have desktop set to folder view ] or when using the desktop menu, easy to miss for since i don't have any icons on the desktop usually. It flickers rapidly

**Comment by bollian** [comment]
I also experience flickering, even when under light load. I'm almost always running Firefox, so I can't verify if the flickering only happens when running a GTK app. I also don't find that switching focused windows changes the flickering, or it only rarely helps rarely, if it does.

The part of the screen that's effected varies. Sometimes it will only be the titlebar, sometimes it will be the contents of the window itself. It happens for both Qt and GTK apps, including Firefox. I've never seen it affect the Plasma Panel, but obviously that part of the screen is fairly static anyway. I've also seen the flickering in both focused and non-focused windows, but never more than one window simultaneously.

I'm currently running master Parachute on mesa 20.0.8 with AMD drivers and Plasma 5.20.4.

**Comment by rago666** [comment]
Can confirm. konsole text flicker, title bar flicker, Sometimes apps won't show up in the overview.

**Comment by miwalter** [comment]
I experienced it (Konsole, Chrome), too. Didn't know it may be related to parachute though - but after disabling it, the (very rare) flickering stopped.

**Comment by tcorreabr** [comment]
Yes, it is a kind of conflict with some open windows. I still have no clue which ones and why.

**Comment by TikaySFO** [comment]
I see the flickering, too. I am on Manjaro KDE Plasma 5.20, X11.

This is the workaround for me: In the Compositor, set "keep window thumbnails" to "never". This is not ideal, but better than not using Parachute.

This is how I experience the flickering: It looks like a graphical glitch, mostly in the vicinity of a moving cursor or close to a menu selection / dropdown selection. It flickers for as long as I move the mouse, if I stop moving the mouse the flickering stops, albeit sometimes with the remaining glitch-artefacts in place. Libreoffice Calc sees it a lot when I select multiple cells, Dolphin is quite affected, too. In Firefox I can have the website jump up and down a row of pixels and back. Sometimes it will not happen for many hours, then it comes back.

**Comment by barolo** [comment]
It's gone for me with the latest Parachute and Plasma

**Comment by bollian** [comment]
I've been using 5.21 for a few days, and the flickering has disappeared for me as well. This is without the thumbnails setting work-around described above.
